scriptable-transform-processor/src/main/java/org/springframework/cloud/stream/module/transform/ScriptableTransformProcessorProperties.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.transform;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for the Scriptable Transform Processor module.
+ *
+ * @author Andy Clement
+ */
+@ConfigurationProperties
+public class ScriptableTransformProcessorProperties {
+
+	/**
+	 * Language of the text in the script property. Supported: groovy, javascript, ruby.
+	 */
+	@NotNull
+	private String language;
```

Comment:
sure thing. I am also still a bit nervous that the other option is called 'script' as anyone with XD experience may think that takes a resource path/url rather than code directly (since that is what script does in other modules).